C++ compile error: missing <cstdint>

<!--
First look if there is already a similar bug report. If there is, upvote the issue with 👍
-->
**Describe the bug**
C++ sdk compilation fails in `component_descriptor.hpp` because `uint64_t` is undefined.

<!-- A clear and concise description of what the bug is. An image is good, a gif or movie is better! -->

**To Reproduce**
1. Create dummy project
CMakeLists.txt
```
cmake_minimum_required(VERSION 3.24)
project(my_test)

add_compile_options(-Wall -Wextra -Wpedantic)

include(FetchContent)
FetchContent_Declare(rerun_sdk URL https://github.com/rerun-io/rerun/releases/download/0.21.0/rerun_cpp_sdk.zip)
FetchContent_MakeAvailable(rerun_sdk)

add_executable(${PROJECT_NAME} dummy.cpp)
target_compile_features(${PROJECT_NAME} PUBLIC cxx_std_17)
target_link_libraries(${PROJECT_NAME} rerun_sdk)
```
dummy.cpp
```
#include <rerun.hpp>
void main() {}
```
2. Build: `cmake . && cmake --build . -j 16`

**Expected behavior**
Builds fine

**Error**
```
cd /home/dherrera/git/test/_deps/rerun_sdk-build && /usr/bin/c++ -DARROW_STATIC -I/home/dherrera/git/test/_deps/rerun_sdk-src/src -isystem /home/dherrera/git/test/arrow/include -Wall -Wextra -Wpedantic -MD -MT _deps/rerun_sdk-build/CMakeFiles/rerun_sdk.dir/src/rerun/blueprint/archetypes/background.cpp.o -MF CMakeFiles/rerun_sdk.dir/src/rerun/blueprint/archetypes/background.cpp.o.d -o CMakeFiles/rerun_sdk.dir/src/rerun/blueprint/archetypes/background.cpp.o -c /home/dherrera/git/test/_deps/rerun_sdk-src/src/rerun/blueprint/archetypes/background.cpp

In file included from /home/dherrera/git/test/_deps/rerun_sdk-src/src/rerun/blueprint/archetypes/../../blueprint/components/background_kind.hpp:6,
                 from /home/dherrera/git/test/_deps/rerun_sdk-src/src/rerun/blueprint/archetypes/background.hpp:6,
                 from /home/dherrera/git/test/_deps/rerun_sdk-src/src/rerun/blueprint/archetypes/background.cpp:4:
/home/dherrera/git/test/_deps/rerun_sdk-src/src/rerun/blueprint/archetypes/../../blueprint/components/../../component_descriptor.hpp:8:37: error: ‘uint64_t’ does not name a type
    8 |     using ComponentDescriptorHash = uint64_t;
      |                                     ^~~~~~~~
/home/dherrera/git/test/_deps/rerun_sdk-src/src/rerun/blueprint/archetypes/../../blueprint/components/../../component_descriptor.hpp:5:1: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
    4 | #include <string_view>
  +++ |+#include <cstdint>
    5 | 
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 24.04
- /usr/bin/c++ --version: c++ (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0

**Rerun version**
0.21.0

--- a/rerun_cpp/src/rerun/component_descriptor.hpp
+++ b/rerun_cpp/src/rerun/component_descriptor.hpp
@@ -2,6 +2,7 @@
 
 #include <optional>
 #include <string_view>
+#include <cstdint>
 
 namespace rerun {
     /// See `ComponentDescriptor::hashed`.

